Blish HUD/GameServices/Content/BitmapFont.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.Xna.Framework.Graphics;
+using MonoGame.Extended.BitmapFonts;
+using System;
+using System.Collections.Generic;
+
+namespace Blish_HUD.Content {
+    /// <summary>
+    /// Extends <see cref="MonoGame.Extended.BitmapFonts.BitmapFont"/> to allow disposing of its glyph lookup texture.
+    /// </summary>
+    public class BitmapFont : MonoGame.Extended.BitmapFonts.BitmapFont, IDisposable {
```

Comment:
Textures of fonts loaded via modules should be disposed when a module unloads.
This will extend Extended.BitmapFont to allow for disposal of the glyph lookup texture.